test/tools/provisioner-cli/main.go

```diff
@@ -97,6 +99,7 @@ func main() {
 
 	if *action == "uploadimage" {
 		log.Info("Uploading PodVM Image...")
```

Comment:
Optional improvement - before stating the `podvmImage` file you could check `podvmImage` is not a zero length string, and error if it is zero length.

The error would be a hint that you need to set the `TEST_PODVM_IMAGE` environment variable when running the image upload action.